Packs/Wiz/Integrations/Wiz/Wiz.yml

```
@@ -227,6 +227,12 @@ script:
       required: true
     - name: reopen_note
       description: reopen note.
+  - name: wiz-get-issue
+    description: Get the details of a Wiz Issue Id.
+    arguments:
+    - name: issue_id
+      description: issue id.
```

It is best practice to write here also how the user can get the issue_id, in that case running the get-issues command.
```suggestion
      description: Issue ID. Get the ID from the "get-issues" command.
```

---

Thanks for the tip. Actually, the only way to get the `Issue ID` is not through the get-issues command, but by copying it from Wiz. We aligned this command format to the other commands that require `issue ID`